README.md

```diff
@@ -1 +1,23 @@
-# jest-fixed-jsdom
+<h1 align="center">jest-fixed-jsdom</h1>
+<p align="center">This library helps re-attach missing NodeJS globals definitions in JSDOM that Jest strips out</p>
+
+## Motivation
+
+Jest strips out a number of NodeJS globals that are used in tests and libraries involving JSDOM, such as structuredClone, ReadableStream, and so on. This library patches these globals back in - there is no polyfilling or mocking involved, it simply re-attaches the missing globals to the JSDOM environment. If you've ever come across errors such as `ReferenceError: ReadableStream is not defined` or `ReferenceError: structuredClone is not defined`, this library is for you. If you were previously using (undici)[https://www.npmjs.com/package/undici] purely to solve this, you will no longer need it. 
```

Comment:
This is not entirely correct. There are a few things that happen in the Jest/JSDOM combo:

- Some globals are stripped away, like `ReadableSteam`, `TextEncoder`, `TextDecoder`.
- Some globals are present but are polyfilled by JSDOM or other libraries, making them incompatible with Node.js, e.g. `Event` and `structuredClone()`. 

If you want, I can help you rewrite this section. Let me know.